### PR TITLE
Fix Broken Links

### PR DESCRIPTION
**Pull request by justindhillon**
### PR fulfills these requirements
- [X] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [x] Tests for the changes have been added/updated (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

### Describe the reason for change
There are many broken links in the documentation of this project. So I have found the correct links and have changed them.

### What has been changed

http://jaywcjlove.github.io/hotkeys --> https://wangchujiang.com/hotkeys-js/

https://docs.launchdarkly.com/sdk/features/flags_from_files#creating_a_flag_data_file --> https://docs.launchdarkly.com/sdk/features/flags-from-files/?q=creating+a+flag+data+file#creating-a-flag-data-file

https://mmdetection.readthedocs.io/en/latest/1_exist_data_model.html#inference-with-existing-models --> https://mmdetection.readthedocs.io/en/latest/user_guides/inference.html

https://github.com/heartexlabs/label-studio/blob/HEAD/label_studio/frontend/src --> https://github.com/HumanSignal/label-studio/tree/develop/label_studio/frontend

http://www.sciencedirect.com/science/article/B6WG4-45JB8WN-9/2/6fe59d8089fd1892c2bfb82283065579 --> https://courses.cs.washington.edu/courses/cse455/09wi/readings/seg_scissors.pdf

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [X] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Support my work
These links were found with [link-inspector](https://github.com/justindhillon/link-inspector). If you find this PR useful, give the repo a ⭐

### CODE REVIEW

**Comment by sajarin** [comment]
/jira create
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/9388624650)
> Jira issue [TRIAG-601](https://humansignal.atlassian.net/browse/TRIAG-601) is created

[TRIAG-601]: https://humansignal.atlassian.net/browse/TRIAG-601?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

File: deploy/default.conf [hunk 1]
```diff
@@ -133,7 +133,7 @@ http {
     }
 
     # LabelStudio frontend
-    # Source: https://github.com/heartexlabs/label-studio/blob/HEAD/label_studio/frontend/src
+    # Source: https://github.com/HumanSignal/label-studio/tree/develop/label_studio/frontend
```
Comment (caitlinwheeless) [inline deploy/default.conf:136]:
```suggestion
    # Source: https://github.com/HumanSignal/label-studio/tree/develop/web/libs/editor
```

**Comment by robot-ci-heartex** [comment]
This PR is stale because it has been open 45 days with no activity. Remove `stale` label or comment or this will be closed in 10 days.

**Comment by robot-ci-heartex** [comment]
This PR is stale because it has been open 45 days with no activity. Remove `stale` label or comment or this will be closed in 10 days.

**Comment by robot-ci-heartex** [comment]
This PR was closed because it has been stalled for 10 days with no activity.

**Pull request closed without merging**